Kickstarter-iOS/Views/Cells/SettingsNotificationCell.swift

```diff
@@ -90,13 +94,12 @@ final class SettingsNotificationCell: UITableViewCell, NibLoading, ValueCell {
   override func bindViewModel() {
     super.bindViewModel()
 
+    self.rac.accessibilityValue = viewModel.outputs.projectCountText
```

Comment:
After testing out the code, I noticed that tapping any cell on `Settings > Notifications` - let's say Project Updates -will make the VoiceOver say `Project Updates, X`, where X is the project count. I may be happening because the `accessibilityValue` is being set to `viewModel.outputs.projectCountText` in every cell here.  I think that this can be solved but returning `projectCountText` only if the cellType = .projectNotifications

Reply:
Good eye! Thanks for catching this!